src/runtime/eval.ml

```diff
@@ -209,6 +209,18 @@ let rec comp {Location.it=c'; at} =
              | Runtime.(Derivation _ | Closure _ | Handler _ | Tag _ | Tuple _ | Ref _ | Dyn _ | String _) as v ->
                 Runtime.(error ~at (JudgementExpected v))
            end)
+  
+  | Syntax.AbstractAtom (a, c) ->
+     comp_as_atom a >>= fun a ->
+           begin comp c >>=
+             function
+             | Runtime.Judgement jdg -> Runtime.return_judgement (Nucleus.abstract_judgement a jdg)
+
+             | Runtime.Boundary bdry -> Runtime.return_boundary (Nucleus.abstract_boundary a bdry)
+
+             | Runtime.(Closure _ | Derivation _| Handler _ | Tag _ | Tuple _ | Ref _ | Dyn _ | String _) as v ->
+                Runtime.(error ~at (JudgementExpected v))
```

Comment:
The error message should be something like `JudgementOrBoundaryExpected`. Do we have one?

Reply:
... in which case the error message above in line 210 is also wrong. Maybe fix that one as well.